nixos/modules/programs/nano.nix

```diff
@@ -42,6 +48,7 @@ in
         include "${cfg.package}/share/nano/*.nanorc"
         include "${cfg.package}/share/nano/extra/*.nanorc"
       '') + cfg.nanorc;
+      variables.EDITOR = lib.mkIf cfg.defaultEditor (lib.mkOverride 900 "nano");
```

Comment:
Isn't `nano` already "the default"?

Reply:
Fair: I think it still makes sense as a feature if you've changed the editor in the past, or just want peace-of-mind that your preferred editor can be set. It also makes nano's options more feature-complete compared to other editors. What do you think about just adding a note that it's the default to the option?